src/resources/authoring/gv2.json

```diff
@@ -426,6 +398,57 @@
         },
         "visibleGenes": "metallic, wings, forelimbs, hindlimbs"
       }
+  },
+
+
+  "levelHierarchy": [
+    // Level 1
```

Comment:
These comments make the file invalid JSON. Depending on how `require` pulls it in, it may make not cause problems now, but it may bite us if we do something like push this into Firebase.